pkg/gds/emails/client.go

```diff
@@ -218,55 +205,49 @@ func (m *EmailManager) SendRejectRegistration(vasp *pb.VASP, reason string) (sen
 		Reason: reason,
 	}
 
-	var contacts = []*pb.Contact{
-		vasp.Contacts.Technical, vasp.Contacts.Administrative,
-		vasp.Contacts.Billing, vasp.Contacts.Legal,
-	}
-
 	// Attempt at least one delivery, don't give up just because one email failed
 	// Track how many emails and errors occurred during delivery.
 	var nErrors uint8
-	for idx, contact := range contacts {
-		// Skip any contacts that we can't send emails to
-		if contact == nil || contact.Email == "" {
+	iter := models.NewContactIterator(vasp.Contacts, false, true)
```

Comment:
Should this be `NewContactIterator(vasp.Contacts, true, true)` -- e.g. both verified contacts and those contacts with emails?